### DISCUSSION TITLE
데이터 학습에 있어서 교환법칙이 성립할까요?

### DISCUSSION THREAD

**Discussion by 20180707jun** [Boostcamp-AI-Tech-1-15/ddobokki_log#24] [1. AI Math]
weight가 똑같이 갱신되어 있다고 치고, A를 학습시키고 B를 학습시킨 weight와 B를 학습시키고 A를 학습시킨 weight가 같은가요?
생각하다보니 네트워크 구조마다 다를 것 같다는 생각은 드네요. mlp, CNN, RNN 에서는 어떤가요?

**Comment by mybirth0407** [comment]
다를 것 같습니다. 학습이란 개념 자체가 bayesian이 아니기 때문인데요, 통계 모델이라면 같을 수도 있지만, 기본적으로는 다를 것 같아요.
그리고 유용한 학습을 위해서는 learning rate가 학습 과정에서 적당히 변경되어야 하는데, 그런 요소들을 포함하면 더욱 더 다를 수밖에 없구요!
